src/templates/main.hbs

```diff
@@ -1,5 +1,7 @@
 {{#each roots key="id" as |root|~}}
   {{~#-in-element root.parent nextSibling=root.nextSibling~}}
-    {{~component root.component~}}
+    {{~#-with-dynamic-vars root=root~}}
```

Comment:
I'd generally prefer if we landed this with the variable name underscored. I don't think we currently consider the dynamic scope system public API at all...